When we remove ghostscript-based filters, CUPS complains about a missing
'gstoraster' filter, even if it doesn't need it. The fix is to drop the
gstoraster rules from the MIME conversion rules. Patch accepted here:

https://bugs.linuxfoundation.org/show_bug.cgi?id=1346

--- COPYING	2016-01-21 14:50:23 +0000
+++ COPYING	2016-03-03 02:35:23 +0000
@@ -103,7 +103,7 @@
   1993-2007, Easy Software Products
 License: GPL-2
 
-Files: mime/cupsfilters.convs*
+Files: mime/cupsfilters*.convs*
 Copyright: 2007-2011, Apple Inc
   1997-2007, Easy Software Products
   2012-2016, Till Kamppeter

--- Makefile.am	2016-03-03 02:39:23 +0000
+++ Makefile.am	2016-03-03 02:44:15 +0000
@@ -292,6 +292,10 @@
 pkgmime_DATA = \
 	mime/cupsfilters.convs \
 	mime/cupsfilters.types
+if ENABLE_GHOSTSCRIPT
+pkgmime_DATA += \
+	mime/cupsfilters-ghostscript.convs
+endif
 if ENABLE_BRAILLE
 pkgmime_DATA += \
 	mime/braille.convs \

--- mime/cupsfilters-ghostscript.convs	1970-01-01 00:00:00 +0000
+++ mime/cupsfilters-ghostscript.convs	2016-03-03 02:31:30 +0000
@@ -0,0 +1,19 @@
+#
+#   MIME conversions file for OpenPrinting CUPS Filters.
+#
+#   Copyright 2007-2011 by Apple Inc.
+#   Copyright 1997-2007 by Easy Software Products.
+#   Copyright 2012-2016 by Till Kamppeter.
+#
+#   These coded instructions, statements, and computer programs are the
+#   property of Apple Inc. and are protected by Federal copyright
+#   law.  Distribution and use rights are outlined in the file "COPYING"
+#   which should have been included with this file.
+#
+
+#
+# Ghostscript-based filters...
+#
+
+application/vnd.cups-pdf	application/vnd.cups-raster	99	gstoraster
+application/vnd.cups-postscript	application/vnd.cups-raster	175	gstoraster

--- mime/cupsfilters.convs	2016-01-21 14:50:23 +0000
+++ mime/cupsfilters.convs	2016-03-03 02:35:59 +0000
@@ -83,8 +83,6 @@
 # Raster filters...
 #
 
-application/vnd.cups-pdf	application/vnd.cups-raster	99	gstoraster
-application/vnd.cups-postscript	application/vnd.cups-raster	175	gstoraster
 application/vnd.cups-pdf	application/vnd.cups-raster	100	pdftoraster
 image/gif			application/vnd.cups-raster	100	imagetoraster
 image/png			application/vnd.cups-raster	100	imagetoraster
